task-management/src/main/java/org/exoplatform/task/management/templates/detail.gtmpl

```diff
@@ -23,8 +24,16 @@
 	def commentCount = taskModel.getCommentCount();
 	def comments = taskModel.getComments();
 	def currentUser = taskModel.getCurrentUser();
-   def labels = taskModel.getLabels();
-   def editableClasses = "hoverStatus editableField editable";
+    def labels = taskModel.getLabels();
+    def editableClasses = "hoverStatus editableField editable";
+    def projectEditableClasses ="hoverStatus ";
+    def projectDataToggle="";
+    def projectInlineEditClass="";
+    if ((task.status == null && task.getCreatedBy() == identity.getUserId())||(task.status != null && task.status.project.canEdit(identity))) {
+        projectEditableClasses = editableClasses;
+        projectDataToggle = "tooltip";
+        projectInlineEditClass = "uiEditableInline";
```

Comment:
The permissions to change the project seem to be only handled by css. What if I have not the permissions but I edit the css classes to be able to edit it ? Is there a check on server side ?